daemon/logger/fluentd/fluentd.go

```diff
@@ -114,8 +162,6 @@ func ValidateLogOpt(cfg map[string]string) error {
 		case "tag":
 		case "labels":
 		case "env":
-		case "fail-on-startup-error":
-		case "buffer-limit":
```

Comment:
Is it ok to just remove these options, without adding options newly added? (`fluentd-buffer-limit`, `fluentd-retry-limit` and others).